genomic_data_service/regulome_atlas.py

```diff
@@ -56,38 +57,73 @@
-FILE_IC_MAX_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_max.bw'
+FILE_IC_MATCHED_MAX_HG19_PATH_LOCAL = './ml_models/bigwig_files/IC_matched_max.bw'
+FILE_IC_MAX_HG19_PATH_LOCAL = './ml_models/bigwig_files/IC_max.bw'
+FILE_IC_MATCHED_MAX_HG19_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_matched_max.bw'
+FILE_IC_MAX_HG19_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_max.bw'
+FILE_IC_MATCHED_MAX_GRCH38_PATH_LOCAL = './ml_models/bigwig_files/IC_matched_max_GRCh38.bw'
+FILE_IC_MAX_GRCH38_PATH_LOCAL = './ml_models/bigwig_files/IC_max_GRCh38.bw'
+FILE_IC_MATCHED_MAX_GRCH38_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_matched_max_GRCh38.bw'
+FILE_IC_MAX_GRCH38_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_max_GRCh38.bw'
 
 try:
     TRAINED_REG_MODEL = pickle.load(
         open('./ml_models/rf_model1.0.1.sav', 'rb'))
 except FileNotFoundError:
     TRAINED_REG_MODEL = None
 
-file_IC_matched_max_exists = exists(FILE_IC_MATCHED_MAX_PATH_LOCAL)
-file_IC_max_exists = exists(FILE_IC_MAX_PATH_LOCAL)
+file_IC_matched_max_hg19_exists = exists(FILE_IC_MATCHED_MAX_HG19_PATH_LOCAL)
+file_IC_max_hg19_exists = exists(FILE_IC_MAX_HG19_PATH_LOCAL)
+file_IC_matched_max_grch38_exists = exists(
+    FILE_IC_MATCHED_MAX_GRCH38_PATH_LOCAL)
+file_IC_max_grch38_exists = exists(FILE_IC_MAX_GRCH38_PATH_LOCAL)
 try:
-    if file_IC_matched_max_exists:
-        IC_MATCHED_MAX_BW = pyBigWig.open(FILE_IC_MATCHED_MAX_PATH_LOCAL)
+    if file_IC_matched_max_hg19_exists:
+        IC_MATCHED_MAX_BW_HG19 = pyBigWig.open(
+            FILE_IC_MATCHED_MAX_HG19_PATH_LOCAL)
     else:
-        IC_MATCHED_MAX_BW = pyBigWig.open(FILE_IC_MATCHED_MAX_PATH_REMOTE)
+        IC_MATCHED_MAX_BW_HG19 = pyBigWig.open(
+            FILE_IC_MATCHED_MAX_HG19_PATH_REMOTE)
 except RuntimeError:
-    IC_MATCHED_MAX_BW = None
+    IC_MATCHED_MAX_BW_HG19 = None
 
 try:
-    if file_IC_max_exists:
-        IC_MAX_BW = pyBigWig.open(FILE_IC_MAX_PATH_LOCAL)
+    if file_IC_max_hg19_exists:
+        IC_MAX_BW_HG19 = pyBigWig.open(FILE_IC_MAX_HG19_PATH_LOCAL)
     else:
-        IC_MAX_BW = pyBigWig.open(FILE_IC_MAX_PATH_REMOTE)
+        IC_MAX_BW_HG19 = pyBigWig.open(FILE_IC_MAX_HG19_PATH_REMOTE)
 except RuntimeError:
-    IC_MAX_BW = None
+    IC_MAX_BW_HG19 = None
+
+try:
+    if file_IC_matched_max_grch38_exists:
+        IC_MATCHED_MAX_BW_GRCH38 = pyBigWig.open(
+            FILE_IC_MATCHED_MAX_GRCH38_PATH_LOCAL)
+    else:
+        IC_MATCHED_MAX_BW_GRCH38 = pyBigWig.open(
+            FILE_IC_MATCHED_MAX_GRCH38_PATH_REMOTE)
+except RuntimeError:
+    IC_MATCHED_MAX_BW_GRCH38 = None
+
+try:
+    if file_IC_max_grch38_exists:
+        IC_MAX_BW_GRCH38 = pyBigWig.open(FILE_IC_MAX_GRCH38_PATH_LOCAL)
+    else:
+        IC_MAX_BW_GRCH38 = pyBigWig.open(FILE_IC_MAX_GRCH38_PATH_REMOTE)
+except RuntimeError:
+    IC_MAX_BW_GRCH38 = None
```

Comment:
Looks like we are repeating the same code here. Could be better to use a function?

Reply:
Sounds good. Will work on it.